composer.json

```diff
@@ -33,7 +33,7 @@
         "php-http/message": "^1.5",
         "psr/http-factory": "^1.0",
         "psr/http-message-implementation": "^1.0",
-        "psr/log": "^1.0",
+        "psr/log": "^1|^2|^3",
```

Comment:
I know it does not matter in this case, but can we stick with the convention of requiring at least the minor version as well?

Reply:
Sure.